examples/transfer_with_timeout.cash

```diff
@@ -13,6 +13,7 @@ contract TransferWithTimeout(
     // Require timeout time to be reached and sender's signature to match
     function timeout(sig senderSig) {
         require(checkSig(senderSig, sender));
+        require(int(tx.version) >= 2);
```

Comment:
After discussing with Licho, we concluded that a tx.version check is never necessary, because it is checked in the protocol. I removed the checks from the Mecenas contract. Could you remove this change from the PR?

Reply:
Done